cmd/githubgen/main.go

```diff
@@ -36,6 +38,8 @@ func main() {
 			generators = append(generators, issueTemplatesGenerator{})
 		case "codeowners":
 			generators = append(generators, codeownersGenerator{})
+		case "distributions":
```

Comment:
Comment above needs an update about what other files we update with this

Reply:
thanks, added